pkg/http/api.go

```diff
@@ -963,3 +975,8 @@ func GetStatusCodeFromMetadata(metadata map[string]string) int {
 
 	return 200
 }
+
+func (a *api) onGetHealthz(c *routing.Context) error {
+	respondEmpty(c.RequestCtx, 200)
```

Comment:
Everything else looks good, this need to respond only when all the components are initialized.